spec.bs

```diff
@@ -2807,21 +2808,19 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=list=] of [=bid debug report
   1. Let |ad| be the [=interest group ad=] from |winningGroup|'s
      [=interest group/ad components=] where the [=interest group ad/render url=]
      equals |componentAd|. If there is no matching element, return failure.
-  1. [=list/Append=] |ad| to |winningAdComponents|.
+  1. [=list/Append=] |ad| onto |winningAdComponents|.
+1. Let |bidValue| be a new [=bid with currency=] with the following [=struct/items=]:
+  : [=bid with currency/value=]
+  :: |response|'s [=server auction response/bid=]

```

Comment:
[=server auction response/bid=] is possible to be null. We need to set it to 0.0001 
 when it's null? https://crsrc.org/c/content/browser/interest_group/interest_group_auction.cc;drc=451093bbaf7fe812bf67d27d760f3bb64c92830b;l=5795